app/retail/templates/settings/email.html

```diff
@@ -41,13 +52,35 @@ <h5>{% trans "Basic Preferences" %}</h5>
         </div>
       </div>
       {% csrf_token %}
-      <input class='button button--primary' type='submit' name='submit' value="Go">
+      <input class='button button--primary' type='submit' name='submit' value="Save">
     </form>
   </div>
 {% endblock %}
 
 {% block scripts %}
 <script>
+    $("document").ready(function(){
+        // select / deselect all
+        $("#select_all").click(function(e){
+            e.preventDefault();
+            if($(this).data('direction') == '+'){
```

Comment:
Do we want to use `===` maybe?

Reply:
Also, here and in the for loop and callback function, I think we should have a space before `{` for better indentation. And also after `if` and `for`

Reply:
Fixing this and the double quotes in the JS blocks.